consumer-features/example-hello-world/v2/example-hello-world-js-jest-pact/tests/consumer/consumer.test.js

```diff
@@ -0,0 +1,48 @@
+// Pact annotated code block - Setting up the Consumer
+const { pactWith } = require("jest-pact");
+const { BearApiClient } = require("../../src/consumer");
+
+pactWith(
+  {
+    consumer: "BearServiceClient",
+    provider: "BearService",
+    dir: "./output/pacts",
+  },
+  (provider) => {
+    let client;
+
+    beforeEach(() => {
+      client = new BearApiClient(provider.mockService.baseUrl);
+    });
+    // End Pact annotated code block
```

Comment:
Hmm, is it going to be clear enough do you think, without the matching end brackets? I can't think how to nicely work round that :thinking: 
Do you think an extra comment like
"// <snip - pactWith block continues>"
would be helpful to at least explain why you're missing the end }); or make it messy?

Reply:
not unless we say the code is for examples and they can c&p from a repo, ideally they are copyable, but that doesn't necessarily work for projects that need several files for scaff, compared to some languages where you get away with everything in one file